### PR TITLE
EBODAC-1088 - adding changes for the new field in Email Export (06-09-2016)

### PR DESCRIPTION
**Pull request by rubenmartin**
This field is used to add nulls in the csv exported only in the export email , the rest of csv should be not afffected

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by rubenmartin** [comment]
@pmuchowski  please add for testing

**Comment by pmuchowski** [comment]
test this please

File: ebodac/src/main/java/org/motechproject/ebodac/service/impl/ExportServiceImpl.java [hunk 1]
```diff
@@ -84,7 +103,36 @@ private void exportEntity(Class<?> entityDtoType, Class<?> entityType, Map<Strin
...
+        try {
+            tableWriter.writeHeader(fields);
+            for (Object entity : entities) {
+                Map<String, String> row = buildRow(entity, headerMap);
+
+                // Forced update due to EBODAC-1088
+                if (showNullsCells) {
+                    for (Entry<String, String> entry : row.entrySet()) {
+                        if (entry.getValue() == null) {
+                            String key = entry.getKey();
```
Comment (pmuchowski) [inline ebodac/src/main/java/org/motechproject/ebodac/service/impl/ExportServiceImpl.java]: @rubenmartin you should use entry.setValue("NULL") instead

**Comment by rubenmartin** [comment]
@pmuchowski  added and tested the input, thanks, please start again the testing .

**Comment by pmuchowski** [comment]
test this please

**Comment by rubenmartin** [comment]
@pmuchowski  thanks

**Pull request merged**